server/middleware/errorHandler.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (err, req, res, next) => {
```

Comment:
This is another location (other than in the server/app.js) where we might try handling errors with a little more distinction between error types - even though this errorhandler from the docs is useful, we could probably provide the client side with more info.  Since we designed the Client and Server in this exercise, it's up to us what the client expects back from the server, and what it does with that info - we could use this to make more explicit errors pop up on the client end, for instance, or use FS to compile a log of information about the error on the server-side to give us a running log of the reqs that cause errors and the sorts of errors they cause.  This would help us track down possible problems by examining that data.  Obviously, this could prove very useful if we had many users connecting to our app and using it every day and we started to get reports from our Customer Support team that something wasn't working for the customer vis-a-vis a certain action.  You could even store that data in a DB, rather than an error log file...